modules/ROOT/pages/about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,689 @@
+
+Following are the YAML tags used to describe the APIs you want to catalog. 
+
+[%header,cols="20a,80a"]
+|===
+| Tag | Description 
+
+| triggerConditions | This section enables you to set conditions to restrict the APIs that are included when the `api-catalog publish-asset` command is run.   
+
+This example sets `triggerConditions` to enable the `api-catalog publish-asset` command to set conditions to publish APIs for just a designated branch or reference, tagged in a certain way, and authored by a specific user. 
+
+---
+ branches:
+   - main
+   - release/(.*)
+ tags:
+   - support
+   - release/(.*)
+ user:
+   - admin
+
+---
+
+| contact | The contact name and email ID.   
+
+| apis | A list of the APIs to be cataloged. 
+
+spec:: The location of the specification file that describes the API. 
+
+assetId:: Specifies the ID of the asset to be published in Exchange. 
+
+docs:: A list of key and value pairs that specify the name of the documentation page and file location for the API.
+These pairs define the order of the documentation pages in Anypoint Platform.
++
+Format: <Name of the documentation page>:<File Location>
++
+If this option is not specified, no documentation pages will be imported. 
+
+categories:: A list of key and value pairs that specify the category name and value. These pairs are validated against the categories that exist in the Anypoint Platform organization.
++
+Format: <Category Name>: <Value(s)>
++
+If this option is not specified, no categories will be added.
+
+customFields:: A list of key and value pairs that specify the field and value. These pairs are validated against the custom fields and field types that exist in the Anypoint Platform organization.
++
+Format: <Field>: <Value(s)> 
++
+If this option is not specified, no custom fields will be added.
+
+versioning:: The asset version. 
++
+If this is not specified, the patch version is incremented by one.
++
+Example: ~1.x.x  
++
+See <<Asset Versioning>>.
+
+apiVersion:: The API version for the asset. 
++
+If the API version is not specified in the descriptor file, the version from the specification file is used. If the API version is specified in both files, the value in the descriptor file is used. The value must be specified in one of the files. 
++
+Example: v1  
+
+|===
+
+=== Asset Versioning
+
+When you publish a version of an asset that already exists in Exchange, the patch version is incremented by default. However, you can set the version using semantic versioning notation `x.x.x` for major, minor, and patch versions respectively. 
+
+Use the following symbols before the version nodes to control how an asset version is incremented:
+
+[%header,cols="15a, 70a"]
+|===
+| Symbol
+
+| Version Node
+
+| ^ | patch
+
+| ~ | minor
+
+| & | major 
+
+Because '&' is a reserved character in YAML, a semantic versioning string using this character must be enclosed in a single quote. For example: '&x.x.x'
+
+|===
+
+==== Examples of Asset Versioning
```

Comment:
yikes, another Level-4

Reply:
These nested heading level issues will be resolved when this page is divided into separate pages (topics) in the GA doc.